code/modules/mob/living/carbon/human/life.dm

```diff
@@ -38,7 +38,7 @@
 		Stun(50)
 
 	if(mind)
-		mind.sleep_adv.add_stress_cycle(get_stress_amount())
+		mind.?.sleep_adv.add_stress_cycle(get_stress_amount())
```

Comment:
```suggestion
		mind.sleep_adv.add_stress_cycle(get_stress_amount())
```
1) `mind.?.sleep_adv` is invalid.
2) We already know `mind` is not null due to `if(mind)`, so `mind?.whatever` is pointless here. If you want to prevent a runtime from having a mind with no `sleep_adv`, change the if to `if(mind?.sleep_adv)`.